DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -78,18 +78,19 @@
 elif(offlineTesting):
     process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
     from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
-    process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run2_data', '')
+    process.GlobalTag = gtCustomise(process.GlobalTag, '100X_dataRun2_v1', '')
```

Comment:
please leave auto:run2_data as global tag in the code, it is more generic and can generally be used with any CMSSW release (tha same apply to the 10_1_X PR)